src/system/RenderableSystem.lua

```diff
@@ -20,6 +20,27 @@ function RenderableSystem:setScene ( tag )
 
 end
 
+function RenderableSystem:update( dt )
```

Comment:
The library I use for animations supports lot's of features such as stop when animation is over, loop back on complete, etc. Implementing that shouldn't cause a lot of troubles. I do recommend to write the game to use how I would refer them as "game ticks" which are in our case probably be either 0.1 or 0.2 seconds. I will open a issue for this.